fairlearn/metrics/_count.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+
+def count(y_true,
+          y_pred,
+          *,
+          sample_weight=None) -> int:
+    """Calculate the number of data points in each group when working with `MetricFrame`.
+
+    The ``y_true`` argument is used to make this calculation. For consistency with
+    other metric functions, the ``y_pred`` argument is required, but ignored.
+
+
+    Parameters
+    ----------
+    y_true : array_like
+        The true labels
+
+    y_pred : array_like
+        The predicted labels (ignored)
+
+    sample_weight : array_like
```

Comment:
If you're going to ignore it, don't include it.

Reply:
I'll remove it. I was trying to be consistent with this: https://github.com/fairlearn/fairlearn/blob/8a76289ed54353872b590c790df8de7bd03ba7e3/fairlearn/metrics/_selection_rate.py#L20-L27